src/client/components/Fetcher/Fetcher.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+
+const baseUrl = '';
+
+async function getUserData(search) {
+  const response = await fetch(`${baseUrl}${encodeURIComponent(search)}`);
+  const result = await response.json();
+  return result;
+}
```

Comment:
Remove.

Reply:
I a little bit confusing, why I should remove it? We just wrote this code with you on Sunday to make fetch reusable.